inspire_utils/name.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, name, constants=None):
             self._parsed_name = HumanName(name, constants=constants)
             self._parsed_name.capitalize()
 
-        if ',' not in name and ('.' not in self.first_list[-1] or len(self.first_list[-1]) == 1):
+        if ',' not in name and (not self.first_list or (self.first_list and '.' not in self.first_list[-1])):
```

Comment:
what about the length thing, don't you need that any longer?

Reply:
I was thinking about the situation when the person is called for example `David Ellis Smith`, where both Ellis and Smith are names but ParsedName would classify David and Ellis as the first names, so then the length condition is no longer valid

Reply:
I don't see what this has to do with it. The `len` test is looking at the length of the last first name, not the number of names.